packages/@aws-cdk/aws-apigateway/package.json

```diff
@@ -82,7 +82,9 @@
   },
   "awslint": {
     "exclude": [
-      "resource-attribute:@aws-cdk/aws-apigateway.IRestApi.restApiRootResourceId"
+      "resource-attribute:@aws-cdk/aws-apigateway.IRestApi.restApiRootResourceId",
+      "resource-interface-extends-construct:@aws-cdk/aws-apigateway.IModel",
```

Comment:
I don’t see a good reason to exclude these

Reply:
This ties into my explanation to @RomainMuller's comment above about the two interfaces for model. The original IModel was added to start getting model support in for MethodResponse, but only going as far as helping to allow the static Empty and Error models to be referenced. I didn't think it necessarily made sense to complicate those model types by making them implement Construct. That said, as above, I can look into what's needed to converge these into one interface. In which case, I will remove this exclusion.

Reply:
I was originally getting build errors on the `IModel` interface, with those complaints about extending construct and including an export() method, but I don't seem to be now. So, I've removed the exclusions. 🤷‍♂️